operations/README.md

```diff
@@ -87,6 +87,9 @@ This section describes all operations supported by the API, organised here by th
 | <div style="width:200px">Operation or Endpoint</div> | Description |
 | :-- | :-- |
 | [Get all loyalty programs](loyaltyprograms.md#get-all-loyalty-programs) | Returns all loyalty programs of the enterprise |
+| [Add loyalty programs](loyaltyprograms.md#add-loyalty-programs) | Adds new loyalty programs and returns details of the added programs |
```

Comment:
We don't really include the `returns details of the added programs` in other add endpoints.